### ISSUE TITLE
ZFS needs a lazytime property to set lazytime mount option

### ISSUE THREAD

**Issue by clinta** [zfsonlinux/zfs#9843]
### System information

Type | Version/Name
 --- | --- 
Distribution Name	| Arch Linux
Linux Kernel	| 5.4.10
Architecture	| x86_64
ZFS Version	| 0.8.2-1
SPL Version	| 0.8.2-1

ZFS supports lazytime, and datasets can be mounted with lazytime by running `mount -o remount,lazytime tank/foo`, but there is no property in the dataset to influence lazytime so that `zfs mount tank/foo` will set lazytime on.

**Comment by smopucilowski** [comment]
Bump? This would be a nice feature to have.

**Comment by clinta** [comment]
I agree, I'd still like to see this added. Unfortunately for me trying to implemented it resulted in a discussion of redesigning the atime interface, and that's a bit beyond my skills.

**Comment by devZer0** [comment]
yes please! and perhaps make lazytime perhaups the default

i found that atime in zfs can cause pathological slowness, so i wonder why an instant read on virtual disk files should hammer the disks with atime updates, slowing don qemu disk image conversion to a crawl

see https://forum.proxmox.com/threads/weird-disk-write-i-o-pattern-on-source-disks-when-moving-virtual-disk.123639/

**Comment by knutov** [comment]
What is the current state of this feature? Is it planned?

Support of LAZYTIME was added to code long time ago - https://github.com/openzfs/zfs/commit/704cd0758a1a4ecef47b0dfa097acd94c8e4f57d

**Comment by marcogabriel** [comment]
Indeed, although it was committed in April 2016 it seems still to be missing from the release. Will there be a chance to see lazytime in zfs?

**Comment by MilesBHuff** [comment]
Bumping for 2025 — this is a clear oversight and seemingly a simple fix. `lazytime` is already supported; it *just* needs a property added that auto-sets it.
